django-stubs/core/cache/backends/base.pyi

```diff
@@ -65,3 +65,5 @@ class BaseCache:
     async def adecr_version(self, key: Any, delta: int = ..., version: int | None = ...) -> int: ...
     def close(self, **kwargs: Any) -> None: ...
     async def aclose(self, **kwargs: Any) -> None: ...
+
+def memcache_key_warnings(key: str) -> Generator[str, None, None]: ...
```

Comment:
```suggestion
def memcache_key_warnings(key: str) -> Iterator[str]: ...
```

Reply:
A question, could you explain to me why the choice for Iterator, and not Generator here?

Reply:
Because `Generator` is only needed when you use `.send` and `.throw` methods. Otherwise, this is just an `Iterator`

Reply:
Although one have to be careful about the not so obvious pitfall found here: https://github.com/python/typeshed/pull/7430

Reply:
I implemented the changes.

Reply:
I guess you didn't push your changes or something, since they didn't appear here. I've just applied the suggestions.